learn/cookbooks/digitalocean_droplet.md

```diff
@@ -1,129 +1,125 @@
 # Deploy a Meilisearch instance on DigitalOcean
 
-[[toc]]
+## Deploy Meilisearch on a DigitalOcean Droplet
```

Comment:
This heading repeats the page title, only without mentioning instance. Do we need to keep it? If yes, can we change the wording so it doesn't feel like the user is reading the same thing twice?

Reply:
I think we should. This page has two sections, the first is on deploying the droplet, and the next is on configuring it. Maybe I should rename the page to avoid duplication.